.eslintrc.json

```diff
@@ -47,6 +47,7 @@
         "block-spacing": "error",
         "array-bracket-spacing": "error",
         "comma-spacing": "error",
+	"spaced-comment": ["error", "always", { "exceptions": ["/"] } ],
```

Comment:
We indent this file using spaces, not tabs. Ironically this is one of the few files that doesn't get linted! Perhaps we should add it to the `eslint:config` task in `Gruntfile.js`.